src/Network/Network/NetworkSecurityPerimeter.Autorest/examples/Remove-AzNetworkSecurityPerimeterAccessRule.md

```diff
@@ -0,0 +1,25 @@
+### Example 1: Deletes a NetworkSecurityPerimeterAccessRule by Name
+
+```powershell
+
+ Remove-AzNetworkSecurityPerimeterAccessRule -Name ar5 -ProfileName profile4 -ResourceGroupName ResourceGroup-1 -SecurityPerimeterName nsp4
+
+```
+
+```output
+
+```
```

Comment:
```suggestion
```
if the command does not have an output the "output" block should be deleted.

Reply:
Removed output where it's empty